repo/zips/plugin.video.nzbdav/index.html

```
@@ -0,0 +1,5 @@
+<html><body>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)